libraries/AP_Math/polygon.cpp

```diff
@@ -88,7 +88,7 @@ bool Polygon_outside(const Vector2<T> &P, const Vector2<T> *V, unsigned n)
 template <typename T>
 bool Polygon_complete(const Vector2<T> *V, unsigned n)
 {
-    return (n >= 4 && V[n-1].x == V[0].x && V[n-1].y == V[0].y);
+    return (n >= 4 && V[n-1] == V[0]);
```

Comment:
OK got it , here is here . not define for int ... https://github.com/ArduPilot/ardupilot/blob/master/libraries/AP_Math/vector2.cpp#L153 .
Then I will switch back to manual use of is_equal :-)